onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -187,6 +192,32 @@ def wrapped(fn):
 
 EXPECTED_SKIPS_OR_FAILS = (
     xfail("add", dtypes=BOOL_TYPES, reason="Add is not defined on bool tensors"),
+    xfail(
+        "acos",
+        dtypes=BOOL_TYPES + INT_TYPES,
+        reason="Acos is not defined on bool or int tensors",
+    ),
+    xfail(
+        "acosh",
+        dtypes=BOOL_TYPES + INT_TYPES,
+        reason="Acosh is not defined on bool or int tensors",
+    ),
+    xfail(
+        "addmm",
+        dtypes=[torch.uint8, torch.int8, torch.int16],
+        reason="MatMul is not defined on int16/int8/uint8 tensors",
+    ),
+    xfail(
```

Comment:
What's the different of 2 "addmm" here?

Reply:
There is a “variant” param which represents a different test. OpInfos are identified by name and variant. The variant is usually empty. When there is a variant OpInfo we need to specify it so that they match.